Example/AnimatedTextInput/ViewController.swift

```diff
@@ -78,6 +78,7 @@ struct CustomTextInputStyle: AnimatedTextInputStyle {
     let lineHeight: CGFloat = 3
     let errorColor = UIColor.red
     let textInputFont = UIFont.systemFont(ofSize: 14)
+    let placeHolderFont = UIFont.systemFont(ofSize: 14)
```

Comment:
Just curious.. Have you tried that the component is working when placeHolderFont.pointSize > textInputFont.pointSize ?

Reply:
It will work, but basically you want input font & placeholder font to be the same.

... Now I start thinking if we ever need to set placeholderFont to anything other than textInputFont.